gordon/metrics/ffwd.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+
+Gordon ships with a simple `ffwd <https://github.com/spotify/ffwd>`_
+metrics implementation, which can be enabled via configuration. This
+module contains the SimpleFfwdRelay, and all required classes that it
+uses to send messsages to the ffwd daemon via UDP.
+
+The `SimpleFfwdRelay` requires no configuration, but can be customized.
+The defaults that may be overridden are shown below.
+
+.. code-block:: ini
+
+    [ffwd]
+    # to identify the service creating metrics
+    key = 'gordon-service'
+
+    # the address of the ffwd daemon (see: UDPClient)
+    ip = '127.0.0.9'
```

Comment:
Nitpick, but toml syntax for strings is double quotes.

Reply:
Ah good point.